staging/src/k8s.io/client-go/testing/fixture.go

```diff
@@ -282,7 +282,7 @@ func (t *tracker) Get(gvr schema.GroupVersionResource, ns, name string) (runtime
 		return nil, errNotFound
 	}
 
-	matchingObjs, err := filterByNamespaceAndName(objs, ns, name)
+	matchingObjs, err := filterByNamespaceAndName(objs, ns, name, false)
```

Comment:
Nit: For readability, could we first assign the boolean to a descriptive variable name? For a reader of the code who doesn't know the signature for `filterByNamespaceAndName`, it can be hard to know what the true/false controls.

Reply:
I moved the check outside of `filterByNamespaceAndName` and directly into `tracket.Get` method instead. So `filterByNamespaceAndName` stays unchanged.